book/security.rst

```diff
@@ -0,0 +1,660 @@
+| *Division*      | ``DEEP_LEVEL``   | This is the same as the *Business Unit* level except that |
+|                 |                  | the user can also access all resources that are owned by  |
+|                 |                  | subordinate units of the business units they are assigned |
+|                 |                  | to.                                                       |
++-----------------+------------------+-----------------------------------------------------------+
+| *Organization*  | ``GLOBAL_LEVEL`` | The user is given access to all records within the        |
+|                 |                  | organization, regardless of the business unit the object  |
+|                 |                  | belongs to or the user is assigned to.                    |
++-----------------+------------------+-----------------------------------------------------------+
+| *System*        | ``SYSTEM_LEVEL`` | The user can access all objects within the system.        |
++-----------------+------------------+-----------------------------------------------------------+
+
+Each record is associated an owning organization. When a user logs into the system, they work in
+the scope of one of their organizations. If a user is member in several organizations, they can
+switch the organization scope that is used to perform access checks.
+
+.. note::
+
+    For all access levels, there is a class constant defined in the
+    :class:`Oro\\Bundle\\SecurityBundle\\Acl\\AccessLevel` class. Its value is shown in the
+    *Constant* column.
+
+    There are two special constants ``AccessLevel::UNKNOWN`` (unknown acccess level, should not be
+    assign to a user) and ``AccessLevel::NONE_LEVEL`` (globally deny access for the user).
+
+.. _permissions:
+
+Permissions
+~~~~~~~~~~~
+
+A user can be assigned different modes to a resource. These modes describe what they are allowed to
+do with the resource. Namely, these permissions are:
+
+==========  ====================================================================
+Permission
+==========  ====================================================================
+``VIEW``    Whether or not a user is allowed to view a record.
+``CREATE``  Whether or not a user is allowed to create a record.
+``EDIT``    Whether or not a user is allowed to modify a record.
+``DELETE``  Whether or not a user is allowed to delete a record.
+``ASSIGN``  Whether or not a user is allowed to assign a record to another user.
+==========  ====================================================================
+
+.. seealso::
+
+    Read the official documentation for a first insight in the `usage of ACLs`_ and more
+    `complex Access Control List`_ examples.
+
+Configuring Entities
+--------------------
+
+To be able to protect the access to your entities, you first have to configure which permissions
+can be granted for a user to them. Use the ``security`` property in the ``defaultValues`` section
```

Comment:
Please change ...``security`` property... to ...``security`` scope... because in termins of entity config, this properties calle
s 'sopes'